assets/src/edit-story/app/story/useStoryReducer/reducers/addPage.js

```diff
@@ -45,12 +51,17 @@ function addPage(state, { page, position }) {
 
   const { id } = page;
 
-  // Ensure new page has elements array and background id
+  // Ensure new page has elements array and background element.
+  const backgroundElementId = uuidv4();
+  const defaultBackgroundElement = {
```

Comment:
Hmm, actually, this logic might be better within `createNewElement`... Checking.